### PR TITLE
login username bug

### LINKED ISSUE #135: the register does not create any Username but login ask for Username
# 🐞 Bug Report

### **Describe the bug**
if no user name is created then how will some one use while login

### **Steps to Reproduce**
2. Click on '...'
3. Scroll down to '...'

### **Expected behavior**
should allow creating username 

### **Screenshots or Video**
Screenshot 2024-10-08 181147
Screenshot 2024-10-08 181202

### **Environment**
Please provide the following details:
- **OS**: Windows
- **Browser/Software**: microsoft edge
- **Version**: _e.g., 22.0_

### PR DESCRIPTION
**Pull request by Kratik1093**
# Pull Request Template for GSoC 2024

## Related Issue #135 

[Please list any related issue numbers that this pull request resolves. If none, state "None"]

closes: #135 

## Description of Changes
CHANGED USER NAME TO NAME IN LOGIN PAGE

## Type of Pull Request

- [X] Bug Fix
- [ ] New Feature
- [ ] Performance Improvement
- [ ] Documentation Update
- [ ] Code Refactoring
- [ ] Security Enhancement
- [ ] Other (Specify): _______________

(Please check relevant boxes by entering an "X" inside them)

## Screenshots / Demo (if applicable)

[Include any screenshots, GIFs, or videos that demonstrate the change or feature, if available.]

## Checklist for Contributor
use'X' in [ ] to check it .

- [X] I have reviewed my code before submission.
- [X] I have read and followed the project's Contribution Guidelines.
- [X] I have tested the code thoroughly.
- [X] I have attached related issue numbers, screenshots, or videos (if applicable).
- [X] I have added comments to my code where necessary for clarity.
- [X] I have checked the existing code base and my changes do not introduce any breaking changes.
- [X] My PR addresses only one issue; I have created separate PRs for other issues.
- [X] I have ensured that the PR does not include unrelated files.
- [X] I am mindful of the project's coding standards and best practices.

## Additional Information

[Include any additional context or relevant information that may help in reviewing this PR.]

## Contribution Guidelines Reminder

Thank you for contributing to our project! Please ensure you follow the guidelines below:

### Contribution Levels

2. **Task Categories:**
   - **Basic Tasks:** Primarily focused on small changes like minor bug fixes, documentation improvements, and unit testing. Labeled as **Easy**.
   - **Intermediate Tasks:** Moderate-level tasks that involve enhancing existing features, code refactoring, or performance improvements. Labeled as **Medium**.
   - **Advanced Tasks:** Complex tasks like building new features, significant refactoring, or critical bug fixes. Labeled as **Hard**.

3. **Acknowledging Effort:**
   - Exceptional contributions will be recognized with a higher contribution level if substantial effort and innovation are demonstrated.

### Review Process

- PRs will be reviewed by the maintainers, and feedback will be provided. Contributors should address feedback promptly and resubmit the PR if necessary.

- Contributors are encouraged to participate in the review process of other PRs for community growth.

We appreciate your time and effort in improving this project and look forward to your valuable contributions!

### CODE REVIEW

**Comment by dhairyagothi** [comment]
image

@Kratik1093 do email here

File: frontend/src/Pages/LoginPage.jsx [hunk 1]
```diff
@@ -38,13 +38,13 @@ const Login = () => {
...
                 {/* Username Input */}
                 <div className="mb-5">
-                    <label className="block mb-2 font-semibold text-gray-700 " htmlFor="username">Username</label>
+                    <label className="block mb-2 font-semibold text-gray-700 " htmlFor="username">Name</label>
                     <input
                         type="text" 
-                        id="username" 
-                        value={username} 
+                        id="name" 
+                        value={name}
```
Comment (dhairyagothi) [inline frontend/src/Pages/LoginPage.jsx]: email

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by Kratik1093** [comment]
pullrequest is been updated with all thereq changes

**Comment by dhairyagothi** [comment]
@Kratik1093 please sync fork there are conflicts 

image

**Pull request closed without merging**